ci-operator/step-registry/openshift-logging/enable-cluster-logging/openshift-logging-enable-cluster-logging-commands.sh

```diff
@@ -0,0 +1,582 @@
+      - effectiveDate: '2023-10-15'
+        version: v13
+    secret:
+      name: s3-secret
+      type: s3
+  storageClassName: ${storage_class}
+  tenants:
+    mode: openshift-logging
+  rules:
+    enabled: true
+    selector:
+      matchLabels:
+        openshift.io/log-alerting: 'true'
+    namespaceSelector:
+      matchLabels:
+        openshift.io/log-alerting: 'true'
+EOF
+}
+
+function wait_lokistack_ready(){
+    echo "### Wait the lokistack ready up to 2 minutes"
+
+    lokistack_ready=false
+    count=1
+    while [[ $count -lt 8 ]]; do
+        if oc -n ${LOKI_NAMESPACE} wait pod --for=condition=Ready  -l app.kubernetes.io/instance=logging-loki; then
+            lokistack_ready=true
+        fi
+        if [[ $lokistack_ready == "true" ]]; then
+            break
+        fi
+        sleep 15s
+        (( count=count + 1 ))
+    done
+
+    if [[ $lokistack_ready == "false" ]]; then
+       echo "lokistack is not ready in 2 minutes"
+       oc -n ${LOKI_NAMESPACE} get pod
+       oc -n ${LOKI_NAMESPACE} get lokistack ${LOKI_NAME} -o json|jq '.status'
+       exit 1
+    fi
+}
+
+function enable_logging_uiplugin(){
+    echo "### Enable_logging_uiplugin"
+    coo=$(oc get csv -o name |grep observability-operator)
+    if [[ $coo == "" ]];then
+        echo "Skip LoggingUI -- no observability-operator"
+    else
+        echo "deploy LoggingUI Plugin"
+        cat <<EOF|oc apply -f -
+apiVersion: observability.openshift.io/v1alpha1
+kind: UIPlugin
+metadata:
+  name: logging
+spec:
+  logging:
+    logsLimit: 20
+    lokiStack:
+      name: ${LOKI_NAME}
+      namespace: ${LOKI_NAMESPACE}
+    schema: select
+    timeout: 5m
+  type: Logging
+EOF
+    fi
+
+}
+
+## deploy clusterloggingforwarder
+function send_logs_to_lokistack()
+{
+echo "### Create obsclf/lfme "
+
+prepare_namespace ${CLF_NAMESPACE}
+oc project ${CLF_NAMESPACE}
+oc create sa logcollector
+oc adm policy add-cluster-role-to-user lokistack-tenant-logs -z logcollector
+oc adm policy add-cluster-role-to-user collect-application-logs -z logcollector
+oc adm policy add-cluster-role-to-user collect-infrastructure-logs -z logcollector
+oc adm policy add-cluster-role-to-user collect-audit-logs -z logcollector
+oc adm policy add-cluster-role-to-user logging-collector-logs-writer  -z logcollector
+
+cat << EOF | oc apply -f -
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: collector
+  namespace: ${CLF_NAMESPACE}
+  annotations:
+    observability.openshift.io/tech-preview-otlp-output: "enabled"
+spec:
+  outputs:
+  - name: my-lokistack
+    lokiStack:
+      dataModel: Viaq
+      authentication:
+        token:
+          from: serviceAccount
+      target:
+        name: ${LOKI_NAME}
+        namespace: ${LOKI_NAMESPACE}
+    tls:
+      ca:
+        key: service-ca.crt
+        configMapName: openshift-service-ca.crt
+    type: lokiStack
+  pipelines:
+  - name: pipe1
+    inputRefs:
+    - infrastructure
+    - application
```

Comment:
Add `audit`?

Reply:
Now. the script can set inputRef base on INPUTS.